features/exacte_match.feature

```diff
@@ -0,0 +1,147 @@
+  En één object (3) met huisnummer=3
+  En huisletter=B
+  En geen huisnummertoevoeging
+  Als er wordt gezocht met postcode=1234AA 
+  En huisnummer=3
+  En geen huisletter
+  En geen huisnummertoevoeging 
+  En exacteMatch=true
+  Dan bevat het resultaat alleen het object (1) met postcode=1234AA 
+  En huisnummer=3
+
+Scenario: combinatie van huisnummer en huisnummertoevoeging
+  Gegeven een verzameling van 5 objecten met postcode=1234AA 
+  En één object (1) met huisnummer=3
+  En geen huisletter
+  En geen huisnummertoevoeging
+  En één object (2) met huisnummer=3
+  En huisletter=A
+  En geen huisnummertoevoeging
+  En één object (3) met huisnummer=3
+  En huisletter=A
+  En huisnummertoevoeging=bis
+  En één object (4) met huisnummer=3
+  En huisletter=B
+  En geen huisnummertoevoeging
+  En één object (5) met huisnummer=3
+  En huisletter=B
+  En huisnummertoevoeging=bis
+  Als er wordt gezocht met postcode=1234AA
+  En huisnummer=3 
+  En huisnummertoevoeging=bis
+  En geen huisletter 
+  En exacteMatch=false
+  Dan bevat het resultaat object (3) met huisnummer=3
+  En huisletter=A 
+  En huisnummertoevoeging=bis 
+  En object (5) met huisnummer=3
+  En huisletter=B
+  En huisnummertoevoeging=bis
+	
+  Gegeven een verzameling van 5 objecten met postcode=1234AA 
+  En één object (1) met huisnummer=3
+  En geen huisletter
+  En geen huisnummertoevoeging
+  En één object (2) met huisnummer=3
+  En huisletter=A
+  En geen huisnummertoevoeging
+  En één object (3) met huisnummer=3
+  En huisletter=A
+  En huisnummertoevoeging=bis
+  En één object (4) met huisnummer=3
+  En huisletter=B
+  En geen huisnummertoevoeging
+  En één object (5) met huisnummer=3
+  En huisletter=B
+  En huisnummertoevoeging=bis
+  Als er wordt gezocht met postcode=1234AA
+  En huisnummer=3 
+  En huisnummertoevoeging=bis
+  En geen huisletter 
+  En exacteMatch=true
+  Dan bevat het resultaat geen objecten
+
+Scenario: combinatie van huisnummer, huisletter en/of huisnummertoevoeging komt meerdere keren voor en exacteMatch wordt niet toegepast
+  Gegeven een verzameling van 2 objecten met postcode=1234AA
+  En huisnummer=3 
+  En geen huisletter 
+  En geen huisnummertoevoeging
+  Als er wordt gezocht met postcode=1234AA
+  En huisnummer=3
+  En geen huisnummertoevoeging
+  En geen huisletter 
+  En exacteMatch=false
+  Dan bevat het resultaat beide objecten
+  
+Scenario: combinatie van huisnummer, huisletter en/of huisnummertoevoeging komt meerdere keren voor en exacteMatch wordt toegepast
+  Gegeven een verzameling van 2 objecten met postcode=1234AA
+  En huisnummer=3 
+  En geen huisletter 
+  En geen huisnummertoevoeging
+  Als er wordt gezocht met postcode=1234AA
+  En huisnummer=3
+  En geen huisnummertoevoeging
+  En geen huisletter 
+  En exacteMatch=true
+  Dan bevat het resultaat beide objecten
+  
+Scenario: endpoints waarbij exacteMatch kan worden toegepast kennen geen fuzzy search
+  Gegeven een verzameling van 2 objecten met postcode=1234AA
```

Comment:
We willen (straks) graag fuzzy search op het adressen endpoint. Dan kan deze scenarionaam verwarrend zijn. Ik zou hier expliciet maken dat dit endpoint geen fuzzy search kent op huisnummer en huisnummertoevoeging.